### ISSUE TITLE
Wallet txs difference

### ISSUE THREAD

**Issue by boorishpea** [mimblewimble/grin-wallet#627]
Recovered the same wallet twice using -t wallet1/ and -t wallet2/

one txs gives this

```
Transaction Log - Account 'default' - Block Height: 1481572
-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
 Id  Type         Shared Transaction Id  Creation Time        TTL Cutoff Height  Confirmed?  Confirmation Time    Num.    Num.     Amount    Amount   Fee   Net         Payment   Kernel  Tx  
                                                                                                                  Inputs  Outputs  Credited  Debited        Difference  Proof             Data 
===============================================================================================================================================================================================
 0   Received Tx  None                   2021-11-13 18:14:38  None               true        2021-11-13 18:14:38  0       1        0.1       0.0      None  0.1         None      None    None 
-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
 1   Received Tx  None                   2021-11-13 18:14:38  None               true        2021-11-13 18:14:38  0       1        0.01      0.0      None  0.01        None      None    None 
-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------


```

the other gives this

```
Transaction Log - Account 'default' - Block Height: 1481573
--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
 Id  Type         Shared Transaction Id                 Creation Time        TTL Cutoff Height  Confirmed?  Confirmation Time    Num.    Num.     Amount    Amount   Fee   Net         Payment   Kernel                                                              Tx  
                                                                                                                                 Inputs  Outputs  Credited  Debited        Difference  Proof                                                                         Data 
==========================================================================================================================================================================================================================================================================
 0   Received Tx  6ee95fe5-f968-4cad-8ffe-19387043ca54  2021-11-06 18:31:12  None               true        2021-11-06 18:48:06  0       1        0.1       0.0      None  0.1         None      0886f447568f22c44541fdbd303eb130f623f352b6fbc4def236ddff2e0b3f7d73  None 
--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
 1   Received Tx  1fc337a1-0539-4a5a-9652-fe232da038dc  2021-11-09 17:15:54  None               true        2021-11-09 17:18:33  0       1        0.01      0.0      None  0.01        None      095540f8c1369d3a34d81fb8a4783a969f73ba09107aee737f1b86c38030aaabce  None 
--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
```
there is clearly some sync issue here, it's the same wallet, txs should show the same info.

**Comment by MCM-Mike** [comment]
>there is clearly some sync issue here, it's the same wallet, txs should show the same info.

Did this also happen after your did a `scan` on both recovered wallets?

`grin-wallet -t wallet1 scan`

**Comment by boorishpea** [comment]
i just tried scan. the shared transaction id's are still showig as None

**Comment by deevope** [comment]
@boorishpea How did you recover those both wallet exactly? Because, we can see that the `Shared Transaction Id` is still present on the other wallet, which means you still had a back up data of the `other wallet`. (.grin/main/wallet_data)

The only thing that i see that would need to be corrected or renamed, is the `Confirmation Time` which represent the latest scan time that detected the output, eg when you recoved your wallet.

For the kernel, as only the outputs are scanned and transactions were aggregated. It cannot appears during the recovery scan.

tl;dr: I don't see any issue. If both of these wallet were recovered without their data back-up, you would end up with the same result apart of `Shared Transaction Id` and `Creation Time` which are timestamp when the scan function detected these outputs.

**Comment by boorishpea** [comment]
sometimes when I restore the amount is available, other times it is not. if what you are saying is that this is just how it is and the transaction history log is not restored when the wallet is restored then ok. an important thing to know.

**Comment by deevope** [comment]
Exactly, some information such as the `Shared Transaction Id` is not restored as it does never appear on the blockchain.
